Firestore/src/FirestoreClient.php

```diff
@@ -325,6 +325,50 @@ public function documents(array $paths, array $options = [])
         );
     }
 
+    /**
+     * Creates a new Query which includes all documents that are contained in
+     * a collection or subcollection with the given collection ID.
+     *
+     * A collection group query with a collection ID of `foo` will return
+     * documents from `/foo` and `/abc/dce/foo`, but not `/foo/bar`.
```

Comment:
Nit: `/foo/bar` is not a collection but a document. You could use `/foo/abc/dce`